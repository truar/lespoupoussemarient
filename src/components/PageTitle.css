.PageTitle h1 {
    display: none;
    font-size: 3rem;
    font-family: SignPainter;
    color: white;
}
.PageTitle {
    display: flex;
    justify-content: center;
    text-align: center;
    height: 56px;
}

@media (min-width: 768px) {
    .PageTitle h1 {
        display: block;
        margin-top: 56px;
        font-size: 4rem;
    }
    .PageTitle {
        height: auto;
    }
}