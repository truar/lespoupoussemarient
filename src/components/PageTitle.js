import React from 'react';
import './PageTitle.css';

function PageTitle(props) {
    return (
        <div className="PageTitle">
            <h1>{props.title}</h1>
        </div>
    );
}

export default PageTitle;