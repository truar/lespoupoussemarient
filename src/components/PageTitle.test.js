import React from 'react';
import PageTitle from './PageTitle';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const component = shallow(<PageTitle />);
  expect(component).toMatchSnapshot();
});
