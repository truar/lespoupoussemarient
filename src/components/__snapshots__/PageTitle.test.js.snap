// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageTitle />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1 />,
      "className": "PageTitle",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1 />,
        "className": "PageTitle",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
